### PR TITLE
Feature/paper rankinf

### PR DESCRIPTION
**Pull request by melisabok**
Create a tag invitation for each group to rank their assigned papers. 

@zbialecki and @celestemartinez I need some help with the instructions 🙏 

Screen Shot 2020-02-18 at 4 22 10 PM

You can also test this working in the dev site:

https://dev.openreview.net/invitation?id=MIDL.io/2020/Conference/Reviewers/-/Paper_Ranking

use: one@google.com, pass: 1234

### CODE REVIEW

File: openreview/conference/templates/paperRankingWebfield.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+  $('#notes .tabs-container').remove();
+
+  Webfield.ui.tabPanel(sections, {
+    container: '#notes',
+    hidden: true
+  });
+
+  // Render the contents of the All Papers tab
+  Webfield.ui.submissionList(notes, {
```
Comment (melisabok) [inline openreview/conference/templates/paperRankingWebfield.js]: @zbialecki Does submissionList has an option to pass the 'referrer' value? I forgot to add that.

Comment (zbialecki) [reply openreview/conference/templates/paperRankingWebfield.js]: No, submissionList currently doesn't support that, but I can add it in my other assignments list page PR.

**Comment by celestemartinez** [comment]
Similar to the instructions MIDL had for ranking:
"Please select how each paper ranks among the entire batch of papers you are reviewing (1=best)"

File: openreview/conference/templates/paperRankingWebfield.js (hunk 1 above)
Comment (zbialecki) [reply openreview/conference/templates/paperRankingWebfield.js]: It's added here: https://github.com/openreview/openreview/pull/1919
You can add the referrer by including it in the `displayOptions` like, 
```
Webfield.ui.submissionList(notes, {
  ...
  displayOptions: { referrer: 'referrer string here' }
}
```

File: openreview/conference/templates/paperRankingWebfield.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+// Add Bid Interface
+// ------------------------------------
+
+var CONFERENCE_ID = '';
+var BLIND_INVITATION_ID = CONFERENCE_ID + '/-/Blind_Submission';
+var ANONREVIEWER_WILDCARD = CONFERENCE_ID + '/Paper.*/AnonReviewer.*';
+var PAPER_RANKING_ID = '';
+var GROUP_NAME = '';
+
+var INSTRUCTIONS = '<p class="dark">Please rank your assigned papers selecting a number in each dropdown below where value 1 is the best.</p>\
```
[the remark this answers is not in the document]
Comment (melisabok) [reply openreview/conference/templates/paperRankingWebfield.js]: I'm not sure if 5 will be always the worst number.

[the remark this answers is not in the document]
Comment (melisabok) [reply openreview/conference/templates/paperRankingWebfield.js]: ok, I put Celeste suggestion.

**Comment by melisabok** [comment]
this should be ready to merge

**Pull request merged**